### ISSUE TITLE
grpc build failure -- unavailable tool: powershell-core

### ISSUE THREAD

**Issue by mellery451** [microsoft/vcpkg#9263]
**Host Environment**
- OS: Windows
- Compiler: vs community 2017

**To Reproduce**
`./vcpkg install grpc`

**Failure logs** 
```
>> vcpkg install grpc
Warning: Different source is available for vcpkg (2019.9.12 -> 0.0.110). Use .\bootstrap-vcpk
g.bat to update.
The following packages will be built and installed:
  * c-ares[core]:x86-windows
    grpc[core]:x86-windows
  * openssl[core]:x86-windows
  * protobuf[core]:x86-windows
  * zlib[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Starting package 1/5: zlib:x86-windows
Building package zlib[core]:x86-windows...
Unknown or unavailable tool: powershell-core
```

**Comment by jozefizso** [comment]
> Warning: Different source is available for vcpkg (2019.9.12 -> 0.0.110). Use .\bootstrap-vcpkg.bat to update.

I think you must recompile vcpkg after updating source code.

**Comment by mellery451** [comment]
this was operator error - turns out I had an older vcpkg dir (with VCPKG_ROOT pointing to that) and it was confusing my newer from-source build. sorry for the noise.
